apps/src/code-studio/pd/components/header.jsx

```diff
@@ -27,14 +27,15 @@ export default class Header extends React.Component {
   renderBreadcrumbItems() {
     const breadcrumbItems = [];
     let builtPath = "/";
-    breadcrumbItems.push({name: "Workshop Dashboard", path: builtPath});
```

Comment:
Rather than remove this and require the caller to supply it, why don't we allow passing the base name as a prop?

See below for more...